layers/+config-files/nginx/packages.el

```diff
@@ -0,0 +1,18 @@
+;;; packages.el --- nginx layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Nathaniel Waisbrot <code@waisbrot.net>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst nginx-packages
+  '(
+    nginx-mode
+    ))
+
+(defun nginx/init-nginx-mode ()
+  (use-package nginx-mode))
```

Comment:
I had no particular reason for not setting `:defer t` beyond a habit of taking the defaults for everything that I don't have a reason to set to a non-default value.